### PR TITLE
Forks Facelift

### PR DESCRIPTION
**Pull request by Soitora**
Holding off on including dev for now unless you want me to included it.
I contacted each fork maintainer and had them give their input until they were satisfied.

### CODE REVIEW

File: src/.vuepress/config/nav.js [hunk 1]
```diff
@@ -15,20 +15,26 @@ module.exports = [
...
-			},
-			{
-				text: "Available extensions",
-				items: [{ text: "List", link: "/extensions/" }]
 			}
 		]
 	},
+	{
+		text: "Forks",
+		link: "/forks/",
```
Comment (arkon) [inline src/.vuepress/config/nav.js]: It isn't really easy to get to this page from the homepage anymore. Might be worth adding a tiny mention to it under the download buttons?

File: src/forks/Neko.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+lang: en-US
+sidebar: false
+---
+
+# <img class="headerLogo" :src="$withBase('/assets/media/fork-Neko-icon.png')"> Neko
+
+<ForkButtons forkName="Neko" downloadLink="https://api.github.com/repos/CarlosEsco/Neko/releases/latest" githubLink="window.open('https://github.com/CarlosEsco/Neko')"/>
+
+## Introduction
+**[Neko](https://github.com/CarlosEsco/Neko)** is a fork based off of **[Tachiyomi](https://github.com/inorichi/tachiyomi)** maintained by **[CarlosEsco](https://github.com/CarlosEsco)**.
```
Comment (arkon) [inline src/forks/Neko.md]: I feel like all of the various links to the main Tachiyomi app/repo are pretty redundant, being on the Tachiyomi website already.

File: src/forks/Neko.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+<img :src="$withBase('/assets/media/fork-Neko-banner.png')"/>
+
+## Features
+::: aside
+For a full list of features, visit the GitHub page.
+:::
+
+- **Features a recommendations tab** <Badge text="Alpha"/>
+- **Built in login with 2FA support**
```
Comment (arkon) [inline src/forks/Neko.md]:
```diff
- Built in
+ Built-in
```

File: src/forks/Neko.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+- **Different theme**
+- **Minor UI changes**
```
Comment (arkon) [inline src/forks/Neko.md]: Are these really that notable? Up to @CarlosEsco , I guess.

File: src/forks/Neko.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+- **Flags for source language in manga info page**
+- **All the mangadex statuses**
```
Comment (arkon) [inline src/forks/Neko.md]:
```diff
- mangadex
+ MangaDex
```

File: src/forks/Neko.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+- **Sort library by date added**
+- **View mdlist follows**
```
Comment (arkon) [inline src/forks/Neko.md]:
```diff
- mdlist
+ MDList
```

File: src/forks/Neko.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+- **Improved Download queue**
+  - This includes reordering and cancelling downloads.
+- **Ability skip chapters that are hidden or read**
```
Comment (arkon) [inline src/forks/Neko.md]:
```diff
- Ability
+ Ability to
```

File: src/forks/TachiyomiEH.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+For a full list of features, visit the GitHub page.
+:::
+
+- **Sync your favorites with your EHentai/ExHentai account**
+- **Search through your favorites by tag/language/artist**
+- **Open galleries in an incognito browser window**
+- **Faster download engine in the reader**
+- **Fingerprint/PIN app lock**
+- **Batch Import gallery URLs**
+  - import single URLs by pasting them into the catalogue search box
```
Comment (arkon) [inline src/forks/TachiyomiEH.md]:
```diff
- import
+ Import
```

File: src/forks/TachiyomiEH.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+::: aside
+For a full list of features, visit the GitHub page.
+:::
+
+- **Sync your favorites with your EHentai/ExHentai account**
+- **Search through your favorites by tag/language/artist**
+- **Open galleries in an incognito browser window**
+- **Faster download engine in the reader**
+- **Fingerprint/PIN app lock**
+- **Batch Import gallery URLs**
```
Comment (arkon) [inline src/forks/TachiyomiEH.md]:
```diff
- Import
+ import
```

File: src/forks/TachiyomiJ2K.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+- **Automatically checks for extension updates**
+- **Reader seekbar overrides switch app gesture in Android 10, so no more accidents for that**
+- **[New material snackbar](https://raw.githubusercontent.com/Jays2Kings/tachiyomi/master/.github/readme-images/material%20snackbar.png)**
+  - Removing manga now auto deletes chapters and has an undo button in case you change your mind.
+- **Option to hide manga titles in grid view**
+- **Library specific enhancements**
+  - Per-category/Drag & drop sorting
+  - Tri-state/Negative filters
+  - Multi-source migration
+- **Webview Navigation**
```
Comment (arkon) [inline src/forks/TachiyomiJ2K.md]: This is in stable too (although it looks a bit different).

File: src/.vuepress/config/nav.js (hunk 1 above)
Comment (Soitora) [reply src/.vuepress/config/nav.js]: You mean under the download buttons in the download modal? I can change that.
"Looking for one of our forks?" ?

File: src/forks/Neko.md (hunk 2 above)
Comment (Soitora) [reply src/forks/Neko.md]: Was initially made as I was gonna make a Dev/Stable page too but I decided not too, I can remove the links but keep the bold.

**Pull request merged**